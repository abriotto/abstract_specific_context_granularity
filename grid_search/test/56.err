  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:01, 65.42it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:00<00:01, 70.64it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:00<00:01, 61.55it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:00<00:01, 59.87it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:00<00:00, 62.56it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:00<00:00, 65.77it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:00<00:00, 57.94it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:00<00:00, 56.22it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:01<00:00, 56.91it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:01<00:00, 64.30it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:01<00:00, 65.03it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:01<00:00, 62.97it/s] 97%|#####################################################################################################################################################################################3     | 96/99 [00:01<00:00, 65.04it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 62.84it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:00, 71.75it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:00<00:00, 51.53it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:00<00:00, 53.87it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 56.10it/s]
