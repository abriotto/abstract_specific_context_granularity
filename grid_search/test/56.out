# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7008844614028931, "acc": 0.5062486529350281, "length": 3.62105393409729, "mode": "train", "epoch": 1}
{"loss": 0.6966350674629211, "acc": 0.5363818407058716, "length": 3.698323965072632, "mode": "test", "epoch": 1}
{"loss": 0.7001118063926697, "acc": 0.5156984925270081, "length": 3.6164183616638184, "mode": "train", "epoch": 2}
{"loss": 0.6992226839065552, "acc": 0.5037243962287903, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6955666542053223, "acc": 0.5006760358810425, "length": 3.5727133750915527, "mode": "train", "epoch": 3}
{"loss": 0.695110559463501, "acc": 0.5000000596046448, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6900240778923035, "acc": 0.5335870385169983, "length": 3.4952685832977295, "mode": "train", "epoch": 4}
{"loss": 0.6951296925544739, "acc": 0.49439167976379395, "length": 1.0055866241455078, "mode": "test", "epoch": 4}
{"loss": 0.6898579001426697, "acc": 0.5252834558486938, "length": 3.4545984268188477, "mode": "train", "epoch": 5}
{"loss": 0.695148229598999, "acc": 0.5236334800720215, "length": 1.251396656036377, "mode": "test", "epoch": 5}
{"loss": 0.6858339905738831, "acc": 0.5364642143249512, "length": 3.3228495121002197, "mode": "train", "epoch": 6}
{"loss": 0.6951707005500793, "acc": 0.5279432535171509, "length": 1.586592197418213, "mode": "test", "epoch": 6}
{"loss": 0.6835419535636902, "acc": 0.550933837890625, "length": 3.2604589462280273, "mode": "train", "epoch": 7}
{"loss": 0.6820732951164246, "acc": 0.5691195130348206, "length": 2.3240222930908203, "mode": "test", "epoch": 7}
{"loss": 0.6676368117332458, "acc": 0.5970867276191711, "length": 3.327306032180786, "mode": "train", "epoch": 8}
{"loss": 0.6528133749961853, "acc": 0.6403854489326477, "length": 3.581005573272705, "mode": "test", "epoch": 8}
{"loss": 0.6383337378501892, "acc": 0.6600860357284546, "length": 3.603888988494873, "mode": "train", "epoch": 9}
{"loss": 0.6237863898277283, "acc": 0.6564246416091919, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6058592200279236, "acc": 0.6907029747962952, "length": 3.815464735031128, "mode": "train", "epoch": 10}
{"loss": 0.5934143662452698, "acc": 0.6806331276893616, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5916029214859009, "acc": 0.6948087811470032, "length": 3.9170775413513184, "mode": "train", "epoch": 11}
{"loss": 0.5840194225311279, "acc": 0.6648044586181641, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.5641862750053406, "acc": 0.6892449259757996, "length": 3.958083152770996, "mode": "train", "epoch": 12}
{"loss": 0.5756888389587402, "acc": 0.7113593220710754, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.577491283416748, "acc": 0.7073714733123779, "length": 3.9652106761932373, "mode": "train", "epoch": 13}
{"loss": 0.5681812167167664, "acc": 0.7076351046562195, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5478504300117493, "acc": 0.6993690729141235, "length": 3.962695837020874, "mode": "train", "epoch": 14}
{"loss": 0.5613938570022583, "acc": 0.7281191349029541, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5412667989730835, "acc": 0.7132474780082703, "length": 3.9624648094177246, "mode": "train", "epoch": 15}
{"loss": 0.550787091255188, "acc": 0.7299814224243164, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5362770557403564, "acc": 0.7280505895614624, "length": 3.970379590988159, "mode": "train", "epoch": 16}
{"loss": 0.5411207675933838, "acc": 0.7476723790168762, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.515835702419281, "acc": 0.728430449962616, "length": 3.9710235595703125, "mode": "train", "epoch": 17}
{"loss": 0.540387749671936, "acc": 0.7420856356620789, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5015417337417603, "acc": 0.7476249933242798, "length": 3.9767801761627197, "mode": "train", "epoch": 18}
{"loss": 0.5217403173446655, "acc": 0.7783983945846558, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.47393184900283813, "acc": 0.7730954885482788, "length": 3.977703809738159, "mode": "train", "epoch": 19}
{"loss": 0.5093477964401245, "acc": 0.7783984541893005, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.46584755182266235, "acc": 0.7885880470275879, "length": 3.9845616817474365, "mode": "train", "epoch": 20}
{"loss": 0.5079267024993896, "acc": 0.7700186967849731, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.4701547622680664, "acc": 0.7986719608306885, "length": 3.989154577255249, "mode": "train", "epoch": 21}
{"loss": 0.516943097114563, "acc": 0.7793295979499817, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.4565611779689789, "acc": 0.8005448579788208, "length": 3.991105556488037, "mode": "train", "epoch": 22}
{"loss": 0.5347878932952881, "acc": 0.7681564092636108, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.44567108154296875, "acc": 0.7980698943138123, "length": 3.992638349533081, "mode": "train", "epoch": 23}
{"loss": 0.49834540486335754, "acc": 0.7849162220954895, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4303772449493408, "acc": 0.8033899068832397, "length": 3.9938933849334717, "mode": "train", "epoch": 24}
{"loss": 0.5056098699569702, "acc": 0.7914338111877441, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.42074838280677795, "acc": 0.803403913974762, "length": 3.992877960205078, "mode": "train", "epoch": 25}
{"loss": 0.5258126258850098, "acc": 0.7914339303970337, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.42016682028770447, "acc": 0.801851212978363, "length": 3.9968831539154053, "mode": "train", "epoch": 26}
{"loss": 0.5277289748191833, "acc": 0.7849162220954895, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.419302374124527, "acc": 0.808982789516449, "length": 3.9946892261505127, "mode": "train", "epoch": 27}
{"loss": 0.5218269228935242, "acc": 0.7858472466468811, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4188075065612793, "acc": 0.8093172907829285, "length": 3.9973108768463135, "mode": "train", "epoch": 28}
{"loss": 0.5370708703994751, "acc": 0.7858472466468811, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4084126055240631, "acc": 0.8018573522567749, "length": 3.9956538677215576, "mode": "train", "epoch": 29}
{"loss": 0.5260224938392639, "acc": 0.782122790813446, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.40786296129226685, "acc": 0.8143054842948914, "length": 3.9967586994171143, "mode": "train", "epoch": 30}
{"loss": 0.5239405632019043, "acc": 0.7867783904075623, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.42207223176956177, "acc": 0.8170360922813416, "length": 3.9947168827056885, "mode": "train", "epoch": 31}
{"loss": 0.5328731536865234, "acc": 0.7728118300437927, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.39271488785743713, "acc": 0.8086755275726318, "length": 3.9962031841278076, "mode": "train", "epoch": 32}
{"loss": 0.5177910327911377, "acc": 0.7830539345741272, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.37205204367637634, "acc": 0.8252077102661133, "length": 3.997972011566162, "mode": "train", "epoch": 33}
{"loss": 0.5280430316925049, "acc": 0.7867782711982727, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.409855455160141, "acc": 0.8210948705673218, "length": 3.994546413421631, "mode": "train", "epoch": 34}
{"loss": 0.5352713465690613, "acc": 0.7774673700332642, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3896256387233734, "acc": 0.8252120018005371, "length": 3.9979910850524902, "mode": "train", "epoch": 35}
{"loss": 0.5214155912399292, "acc": 0.7783985733985901, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.37336263060569763, "acc": 0.8323646187782288, "length": 3.99758243560791, "mode": "train", "epoch": 36}
{"loss": 0.5149531364440918, "acc": 0.7914339900016785, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3715159296989441, "acc": 0.834868848323822, "length": 3.99853777885437, "mode": "train", "epoch": 37}
{"loss": 0.5083880424499512, "acc": 0.7783986330032349, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3462100327014923, "acc": 0.8529358506202698, "length": 3.998884677886963, "mode": "train", "epoch": 38}
{"loss": 0.5219971537590027, "acc": 0.7858473062515259, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3517955243587494, "acc": 0.8429718017578125, "length": 3.998730421066284, "mode": "train", "epoch": 39}
{"loss": 0.5275580883026123, "acc": 0.7849161028862, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3357286751270294, "acc": 0.8526378870010376, "length": 3.999180316925049, "mode": "train", "epoch": 40}
{"loss": 0.5173777937889099, "acc": 0.7737430334091187, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.35253292322158813, "acc": 0.8545055389404297, "length": 3.9993579387664795, "mode": "train", "epoch": 41}
{"loss": 0.5126815438270569, "acc": 0.7802606225013733, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.33498844504356384, "acc": 0.8560583591461182, "length": 3.9994122982025146, "mode": "train", "epoch": 42}
{"loss": 0.5051256418228149, "acc": 0.7877094149589539, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.33221888542175293, "acc": 0.8619837760925293, "length": 3.999661445617676, "mode": "train", "epoch": 43}
{"loss": 0.5214371681213379, "acc": 0.7802606225013733, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3399059772491455, "acc": 0.8575484156608582, "length": 3.996162176132202, "mode": "train", "epoch": 44}
{"loss": 0.5130618214607239, "acc": 0.7895717620849609, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.31997451186180115, "acc": 0.8672385811805725, "length": 3.997774839401245, "mode": "train", "epoch": 45}
{"loss": 0.4981854259967804, "acc": 0.7783984541893005, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3245844542980194, "acc": 0.860429584980011, "length": 3.9997212886810303, "mode": "train", "epoch": 46}
{"loss": 0.49547645449638367, "acc": 0.7979514598846436, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3143206238746643, "acc": 0.8613690733909607, "length": 3.999948501586914, "mode": "train", "epoch": 47}
{"loss": 0.49612152576446533, "acc": 0.7905027270317078, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.32040783762931824, "acc": 0.8631967902183533, "length": 3.998483657836914, "mode": "train", "epoch": 48}
{"loss": 0.5138714909553528, "acc": 0.7774673700332642, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.32918083667755127, "acc": 0.8644843697547913, "length": 3.999912738800049, "mode": "train", "epoch": 49}
{"loss": 0.5162457823753357, "acc": 0.7802606225013733, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3160901963710785, "acc": 0.8686432242393494, "length": 3.9944136142730713, "mode": "train", "epoch": 50}
{"loss": 0.507290244102478, "acc": 0.7793295383453369, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3095179498195648, "acc": 0.868532121181488, "length": 3.9998745918273926, "mode": "train", "epoch": 51}
{"loss": 0.5082564949989319, "acc": 0.7830539345741272, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.31444454193115234, "acc": 0.8741370439529419, "length": 3.9997973442077637, "mode": "train", "epoch": 52}
{"loss": 0.515121579170227, "acc": 0.7905027270317078, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3147551715373993, "acc": 0.8678823113441467, "length": 3.99927020072937, "mode": "train", "epoch": 53}
{"loss": 0.5146341919898987, "acc": 0.7849162817001343, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3306570053100586, "acc": 0.8588769435882568, "length": 3.9998960494995117, "mode": "train", "epoch": 54}
{"loss": 0.5090160369873047, "acc": 0.7960894107818604, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.3028051555156708, "acc": 0.8688349723815918, "length": 3.9995760917663574, "mode": "train", "epoch": 55}
{"loss": 0.5001603364944458, "acc": 0.795158326625824, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3234364986419678, "acc": 0.8725302219390869, "length": 3.99849009513855, "mode": "train", "epoch": 56}
{"loss": 0.5095872282981873, "acc": 0.8007449507713318, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.30414822697639465, "acc": 0.8700903058052063, "length": 3.9998767375946045, "mode": "train", "epoch": 57}
{"loss": 0.5218459367752075, "acc": 0.8016759157180786, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2956937551498413, "acc": 0.8728947639465332, "length": 3.999906063079834, "mode": "train", "epoch": 58}
{"loss": 0.5129636526107788, "acc": 0.8072625994682312, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3167286813259125, "acc": 0.8694695234298706, "length": 3.9999606609344482, "mode": "train", "epoch": 59}
{"loss": 0.49121639132499695, "acc": 0.8091248273849487, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.30438363552093506, "acc": 0.8746423721313477, "length": 3.997103214263916, "mode": "train", "epoch": 60}
{"loss": 0.5031223893165588, "acc": 0.8063315153121948, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.3019025921821594, "acc": 0.8672823905944824, "length": 3.9996349811553955, "mode": "train", "epoch": 61}
{"loss": 0.49428921937942505, "acc": 0.8044691681861877, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.29163312911987305, "acc": 0.8760115504264832, "length": 3.999966621398926, "mode": "train", "epoch": 62}
{"loss": 0.49938908219337463, "acc": 0.7839851379394531, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.28831902146339417, "acc": 0.8728830814361572, "length": 3.9990243911743164, "mode": "train", "epoch": 63}
{"loss": 0.512719988822937, "acc": 0.8072624802589417, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.28575336933135986, "acc": 0.88130784034729, "length": 3.999981641769409, "mode": "train", "epoch": 64}
{"loss": 0.49643972516059875, "acc": 0.8119181394577026, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.2997102737426758, "acc": 0.8859797120094299, "length": 3.9999420642852783, "mode": "train", "epoch": 65}
{"loss": 0.49665266275405884, "acc": 0.8072624802589417, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.2767634093761444, "acc": 0.8934563994407654, "length": 3.9999287128448486, "mode": "train", "epoch": 66}
{"loss": 0.49174386262893677, "acc": 0.8184357285499573, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.27976855635643005, "acc": 0.8822392821311951, "length": 3.999852180480957, "mode": "train", "epoch": 67}
{"loss": 0.4952392578125, "acc": 0.8128491044044495, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.2880932688713074, "acc": 0.885981023311615, "length": 3.9999899864196777, "mode": "train", "epoch": 68}
{"loss": 0.4980892539024353, "acc": 0.8109870553016663, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.2698834240436554, "acc": 0.8920638561248779, "length": 3.9965574741363525, "mode": "train", "epoch": 69}
{"loss": 0.5141454935073853, "acc": 0.8137801885604858, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.2740418612957001, "acc": 0.888158917427063, "length": 3.999922513961792, "mode": "train", "epoch": 70}
{"loss": 0.4838992953300476, "acc": 0.8128491640090942, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.2723982036113739, "acc": 0.8906524777412415, "length": 3.99993896484375, "mode": "train", "epoch": 71}
{"loss": 0.47920042276382446, "acc": 0.8137801885604858, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.2566440999507904, "acc": 0.8909644484519958, "length": 3.999967575073242, "mode": "train", "epoch": 72}
{"loss": 0.47780582308769226, "acc": 0.8109869360923767, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.29535171389579773, "acc": 0.8875364661216736, "length": 3.999938488006592, "mode": "train", "epoch": 73}
{"loss": 0.47819867730140686, "acc": 0.8100558519363403, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.27991804480552673, "acc": 0.8950133323669434, "length": 3.999953508377075, "mode": "train", "epoch": 74}
{"loss": 0.48090624809265137, "acc": 0.8193668127059937, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.24710892140865326, "acc": 0.8996866345405579, "length": 3.999936103820801, "mode": "train", "epoch": 75}
{"loss": 0.500829815864563, "acc": 0.8258845210075378, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.2553088366985321, "acc": 0.8937562704086304, "length": 3.9996984004974365, "mode": "train", "epoch": 76}
{"loss": 0.48599493503570557, "acc": 0.8109868168830872, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.27254170179367065, "acc": 0.9006215333938599, "length": 3.9999566078186035, "mode": "train", "epoch": 77}
{"loss": 0.4919147491455078, "acc": 0.8249533772468567, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.2755109369754791, "acc": 0.8943890333175659, "length": 3.9999241828918457, "mode": "train", "epoch": 78}
{"loss": 0.4842454791069031, "acc": 0.8175048232078552, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.24481435120105743, "acc": 0.896572470664978, "length": 3.999976634979248, "mode": "train", "epoch": 79}
{"loss": 0.46715664863586426, "acc": 0.8286778330802917, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.25871264934539795, "acc": 0.8990640044212341, "length": 3.9999465942382812, "mode": "train", "epoch": 80}
{"loss": 0.48183050751686096, "acc": 0.826815664768219, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.2530268430709839, "acc": 0.897091269493103, "length": 3.996339797973633, "mode": "train", "epoch": 81}
{"loss": 0.46480411291122437, "acc": 0.8258845806121826, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.25038766860961914, "acc": 0.898987352848053, "length": 3.997565507888794, "mode": "train", "epoch": 82}
{"loss": 0.47698742151260376, "acc": 0.826815664768219, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.2624730169773102, "acc": 0.8968546986579895, "length": 3.99893856048584, "mode": "train", "epoch": 83}
{"loss": 0.45776861906051636, "acc": 0.8342645168304443, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.2452658861875534, "acc": 0.9015342593193054, "length": 3.99940824508667, "mode": "train", "epoch": 84}
{"loss": 0.48126304149627686, "acc": 0.8305400013923645, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.2333739846944809, "acc": 0.9008980393409729, "length": 3.998870611190796, "mode": "train", "epoch": 85}
{"loss": 0.4874367415904999, "acc": 0.8277466893196106, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.24649818241596222, "acc": 0.9006184935569763, "length": 3.999767780303955, "mode": "train", "epoch": 86}
{"loss": 0.4783950448036194, "acc": 0.8314710855484009, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.23198924958705902, "acc": 0.9024911522865295, "length": 3.9947776794433594, "mode": "train", "epoch": 87}
{"loss": 0.49380937218666077, "acc": 0.8286778926849365, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.2390589714050293, "acc": 0.9031139612197876, "length": 3.9999330043792725, "mode": "train", "epoch": 88}
{"loss": 0.5353230237960815, "acc": 0.8277466893196106, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.22490492463111877, "acc": 0.901245653629303, "length": 3.9999866485595703, "mode": "train", "epoch": 89}
{"loss": 0.5000423789024353, "acc": 0.8351954221725464, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.23481455445289612, "acc": 0.9018674492835999, "length": 3.9999425411224365, "mode": "train", "epoch": 90}
{"loss": 0.4832286238670349, "acc": 0.8286778926849365, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.2285674810409546, "acc": 0.9112144708633423, "length": 3.9999918937683105, "mode": "train", "epoch": 91}
{"loss": 0.4949952960014343, "acc": 0.8268157243728638, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.22403214871883392, "acc": 0.9130790829658508, "length": 3.9999051094055176, "mode": "train", "epoch": 92}
{"loss": 0.5059289932250977, "acc": 0.8370577096939087, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.22813455760478973, "acc": 0.9142061471939087, "length": 3.9977352619171143, "mode": "train", "epoch": 93}
{"loss": 0.48802703619003296, "acc": 0.8379888534545898, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.22493350505828857, "acc": 0.9139760136604309, "length": 3.9991726875305176, "mode": "train", "epoch": 94}
{"loss": 0.4860231280326843, "acc": 0.83426433801651, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.23538784682750702, "acc": 0.9118375182151794, "length": 3.99997615814209, "mode": "train", "epoch": 95}
{"loss": 0.49332645535469055, "acc": 0.839850902557373, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.23589543998241425, "acc": 0.9118375778198242, "length": 3.999992609024048, "mode": "train", "epoch": 96}
{"loss": 0.4921605587005615, "acc": 0.8435754776000977, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.22205737233161926, "acc": 0.9121487140655518, "length": 3.99997615814209, "mode": "train", "epoch": 97}
{"loss": 0.46791672706604004, "acc": 0.8417132496833801, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.21534906327724457, "acc": 0.910591721534729, "length": 3.999988079071045, "mode": "train", "epoch": 98}
{"loss": 0.47314685583114624, "acc": 0.8435753583908081, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.22413110733032227, "acc": 0.9137070775032043, "length": 3.999992609024048, "mode": "train", "epoch": 99}
{"loss": 0.4728052020072937, "acc": 0.8389200568199158, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.2239769995212555, "acc": 0.9143301248550415, "length": 3.9999639987945557, "mode": "train", "epoch": 100}
{"loss": 0.5097681879997253, "acc": 0.8351954221725464, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7777777910232544
